### PR TITLE
[FEATURE] Update RBP notebook to include example for Column-Pair Domain Builder

### PR DESCRIPTION
**Pull request by Shinnnyshinshin**
Changes proposed in this pull request:
- Follow-up PR to #4608 
- Include working example for RBP notebook for Column-Pair Domain Builder

### Definition of Done
Please delete options that are not relevant.

- [x] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style)
- [x] I have performed a [self-review](https://docs.greatexpectations.io/docs/contributing/contributing_checklist) of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have run any local integration tests and made sure that nothing is broken.

Thank you for submitting!

### CODE REVIEW

File: tests/test_fixtures/rule_based_profiler/example_notebooks/BasicExample_RBP_Instantiation_and_running.ipynb [hunk 1]
```diff
@@ -1328,6 +1328,66 @@
...
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**ColumnPairDomainBuilder**\n",
+    "\n",
+    "This DomainBuilder outputs `ColumnPair` Domains by taking in a column pair list in the `include_column_names` parameter."
+   ]
+  },
```
Comment (Shinnnyshinshin) [inline tests/test_fixtures/rule_based_profiler/example_notebooks/BasicExample_RBP_Instantiation_and_running.ipynb]: This is the key change

File: tests/test_fixtures/rule_based_profiler/example_notebooks/BasicExample_RBP_Instantiation_and_running.ipynb [hunk 2]
```diff
@@ -1328,6 +1328,66 @@
...
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**ColumnPairDomainBuilder**\n",
+    "\n",
+    "This DomainBuilder outputs `columnpair` domains by taking in a column pair list in the `include_column_names` parameter."
+   ]
+  },
```
Comment (Shinnnyshinshin) [inline tests/test_fixtures/rule_based_profiler/example_notebooks/BasicExample_RBP_Instantiation_and_running.ipynb:1339]: This is the section that is added to the Appendix

File: tests/test_fixtures/rule_based_profiler/example_notebooks/BasicExample_RBP_Instantiation_and_running.ipynb [hunk 3]
```diff
@@ -1328,6 +1328,66 @@
...
+   "source": [
+    "domain_builder: DomainBuilder = ColumnPairDomainBuilder(\n",
+    "    include_column_names=[\"vendor_id\", \"fare_amount\"],\n",
+    "    batch_request=single_batch_batch_request,\n",
+    "    data_context=data_context,\n",
+    ")\n",
+    "domains: list = domain_builder.get_domains()\n",
+    "assert len(domains) == 1 # 2 columns are part of a single columnpair domain. "
+   ]
+  },
```
Comment (Shinnnyshinshin) [inline tests/test_fixtures/rule_based_profiler/example_notebooks/BasicExample_RBP_Instantiation_and_running.ipynb:1372]: And here is the code that exercises the new `ColumnPairDomainBuilder`

**Review by alexsherstinsky — APPROVED** [review]
LGTM

**Comment by Shinnnyshinshin** [comment]
closing because these changes were added to #4704

**Pull request closed without merging**